lib/cron.js

```diff
@@ -93,8 +97,11 @@ class JobManager {
         } = jobType;
 
         if (moment().isAfter(time)) {
-            logger.warn(`Job "${description}" with params %o is not added: \
-is in the past (${moment(time).format('YYYY-MM-DD HH:mm:SS')}), not scheduling.`, params);
+            logger.warn({
+                description,
+                scheduled_on: moment(time).format('YYYY-MM-DD HH:mm:SS'),
```

Comment:
we don't need any timestamp i presume, tbh - the moment when ES receives the log should be the same that `moment` is telling us (for our case, the approximation of ignoring the transmission time of the message is enough imho)

Reply:
It's not the timestamp, it's the time a job will be executed at